docs/logs/correlation/Program.cs

```diff
@@ -34,8 +36,10 @@ public static void Main()
             logger.FoodPriceChanged("artichoke", 9.99);
         }
 
-        // Dispose logger factory before the application ends.
-        // This will flush the remaining logs and shutdown the logging pipeline.
+        // This will flush the remaining logs.
```

Comment:
This will not flush the remaining logs, it will try to flush and that could fail.

Reply:
Attempted prose additions around flush failure in 'extending the sdk' project.